<launch>

    <arg name="leaf_size" default="0.05"/>    
    <arg name="noise_filter_threshold" default="0.05"/>    
    <arg name="floor_filter_threshold" default="0.05"/>    

    <node name="filter" pkg="leica_scanstation" type="filter" output="screen" ns="cad">
        <param name="leaf_size"                value="0.05"/>
        <param name="noise_filter_threshold"   value="-1"/>
        <param name="floor_filter_threshold"   value="-1"/>
    </node>

    <!-- <node name="filter" pkg="leica_scanstation" type="filter" output="screen" ns="scan">
        <param name="leaf_size"                value="0.05"/>
        <param name="noise_filter_threshold"   value="5"/>
        <param name="floor_filter_threshold"   value="0.8"/>
    </node> -->


</launch>